galleon-pack/src/main/resources/modules/system/layers/base/org/wildfly/extension/elytron-tls/main/module.xml

```diff
@@ -0,0 +1,42 @@
+<!--
+  ~ Copyright 2019 Red Hat, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<module xmlns="urn:jboss:module:1.9" name="org.wildfly.extension.elytron-tls">
+    <properties>
+        <property name="jboss.api" value="private"/>
+    </properties>
+
+    <resources>
+        <artifact name="${org.wildfly.extras.elytron-tls:elytron-tls-subsystem}"/>
+        <artifact name="${org.wildfly.security:wildfly-elytron-ssl-common}"/>
+    </resources>
+
+    <dependencies>
+        <!-- Typical module dependencies for a subsystem -->
+        <module name="java.security.sasl"/>
```

Comment:
Just total minor, I would think that the java.security.sasl is not needed here for TLS subsystem

Reply:
yeah, I was talking to Farah about it last week, I'll take it out after I check if anything needs it